server/src-lib/Hasura/GraphQL/Context.hs

```diff
@@ -197,7 +206,10 @@ mkCompExpInp colTy =
   , map (mk $ G.toLT colScalarTy) listOps
   , bool [] (map (mk $ mkScalarTy PGText) stringOps) isStringTy
   , bool [] (map jsonbOpToInpVal jsonbOps) isJsonbTy
-  , bool [] (stDWithinOpInpVal : map geomOpToInpVal geomOps) isGeometryTy
+  , bool [] (stDWithinGeoOpInpVal stDWithinGeoDesc stDWithinGeometryInpTy :
+             map geoOpToInpVal (geoOps ++ geomOps)) isGeometryType
+  , bool [] (stDWithinGeoOpInpVal stDWithinGeoDesc stDWithinGeographyInpTy :
```

Comment:
passing `stDWithinGeoDesc` is not required. You directly use it in function implementation.

Reply:
Done